ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
Right, let's wait for @domdom82 or @lehoanganh to chime in, they'll know.

Can we add at least a linecomment for that variable in `group_vars/all` to describe what it does? Something like

```
# The user to install docker for. Defaults to the ansible user iff not set. This will be the 
# user who is able to run docker commands on a machine setup with prereq_build.yml
```